Simulator/conversions.py

```diff
@@ -84,16 +84,15 @@ def NED_to_ECI(vec_NED, glat, glon, GMST):
     return vec_ECI
 
 
-
 #--------------------Quaternions-----------------------------
 
 def L(q):
```

Comment:
Even though this is on your pull request Nick, this is actually a @hridujain thing according to the git blame:

Allocating `Vhat = skew(v)` and then using values from `Vhat` is very silly, since the values of `Vhat` are all _already_ in `v`. This allocates an unnecessary numpy matrix which as we now know carries sizable overhead. In particular:
```python
Vhat[0, 1] = -v[2]
Vhat[0, 2] = v[1]
Vhat[1, 2] = -v[0]
```
And the transposed values are negative since it's skew-symmetric. This comes up in both `L` and `R`; would you mind changing it?

Reply:
Do you want me or @hridujain to change it? I think the implementation is correct, but you're right it's inefficient. Are you saying you think it gives the wrong answer too?

Reply:
No, I believe the answer is correct, just that the implementation is inefficient (R and L each allocate an unnecessary numpy matrix).

Reply:
@tomerarnon It turns out that this was actually a change Paul and/or Ethan made that I later received credit for due to that annoying line endings issue. : [link to commit](https://github.com/spacecraft-design-lab-2019/groundtruth-simulator/commit/acc75dda1b45e8fec05c72af7148fa52b03279a4)

I made several tests at the time and noted that the implementations were identical. Here is the link to the discussion I had with Ethan regarding this: [link](https://github.com/spacecraft-design-lab-2019/groundtruth-simulator/pull/44#discussion_r351003056)

TLDR the change was made because numpy is stupid slow. But I agree that we can improve it further exactly as you suggested @tomerarnon

Reply:
This was done in the validation branch, so resolving here.